dracut/30ignition/ignition-quench.service

```diff
@@ -1,16 +1,16 @@
 [Unit]
 Description=Ignition (record completion)
 DefaultDependencies=false
-Before=initrd.target
+#Before=initrd.target
```

Comment:
Do we need this unit at all anymore then?

Reply:
I think we do still need this unit, but we can probably change it to just use `ConditionKernelCommandLine=` and not depend on ignition at all right?

Reply:
Tangentially related: I am actually a bit confused as to having the `.clc`, `.ign`, *and* the split out `.service` file for this.

Reply:
The .clc is generated by the .ign and these are Derek's attempt at getting ignition to run on the first boot. He told me to ignore these files.

Reply:
yep. we should be able to delete those files as part of this PR (if I understand correctly). the .ign was generated from the .clc (i asked him to add the .clc for transparency).

Reply:
Deleted!